configs/community.dbuild

```diff
@@ -185,17 +185,10 @@ build += {
     ]
   }
 
-  // forked (October 2018) for JDK 11 friendliness
-  // frozen (February 2019) at a December 2018 commit
-  // because of a downstream circe failure, reported here:
-  // https://github.com/circe/circe/issues/1084
   ${vars.base} {
     name: "scalacheck"
     uri:  ${vars.uris.scalacheck-uri}
     extra.projects: ["jvm"]  // no Scala.js please
-    extra.commands: ${vars.base.extra.commands} [
-      "set jvm/Test/unmanagedSources/excludeFilter := HiddenFileFilter || \"SerializabilitySpecification.scala\""
-    ]
```

Comment:
There isn't any evidence that this test is disabled.  Appears to run, and run successfully.

Reply:
I think the way we filter out some tests stopped working in sbt 1.3.0-RCx. I'm going to dig into it later, but anyway, good to know that we don't need to filter the test anymore.